docs/modules/ROOT/pages/getting-started.adoc

```diff
@@ -1 +1,162 @@
+        options: solidityLoaderOptions,
+      },
+    ],
+  },
+  solidityLoaderOptions,
+};
+----
+
+== Counter contract
+Compile the Counter contract with OpenZeppelin CLI.
+
+[source,console]
+----
+$ npx oz compile
+✓ Compiled contracts with solc 0.5.13 (commit.5b0b510c)
+----
+
+Deploy the Counter contract to our local blockchain.
+
+Run `npx oz create` and use the interactive commands to deploy the Counter contract to the development network and initialize to a value e.g. `23`.
+
+[source,console]
+----
+$ npx oz create
+Nothing to compile, all contracts are up to date.
+? Pick a contract to instantiate Counter
+? Pick a network development
+✓ Added contract Counter
+✓ Contract Counter deployed
+All contracts have been deployed
+? Call a function to initialize the instance after creating it? Yes
+? Select which function * initialize(num: uint256)
+? num (uint256): 23
+✓ Setting everything up to create contract instances
+✓ Instance created at 0xCfEB869F69431e42cdB54A4F4f105C19C080A601
+0xCfEB869F69431e42cdB54A4F4f105C19C080A601
+----
+
+== App
+We will now start the app.
+
+[source,console]
+----
+$ cd client
+$ npm run start
+----
+
+The app will open in a browser on http://localhost:3000/
+
+Click on Counter to go to http://localhost:3000/counter
+
+In MetaMask change your network to Localhost:8545 to connect to `ganache-cli`.
+
+== Fund MetaMask account
+If you need to fund your MetaMask account (e.g. it has a balance of 0 test Ether).
+
+From your project directory in another terminal, call `npx oz transfer` to transfer 0.5 (test) ether to your account in MetaMask.
+
+[source,console]
+----
+$ npx oz transfer
+? Pick a network development
+? Choose the account to send transactions from (0) 0xF983502f33E2c9E62d0465557446B86C595C4919
+? Enter the receiver account 0x77737a65C296012C67F8c7f656d1Df81827c9541
+? Enter an amount to transfer 0.5 ether
+✓ Funds sent. Transaction hash: 0x69a0ecc17b325a0e4cd414ca9d197a59e88a466c8d3b4d7a0a518ca384526af4
+----
+
+== Use the app
+In the app, press *Increase Counter by 1* button. Confirm the transaction in MetaMask and once the transaction is confirmed the app will have the updated counter value.
+
+== Update the Counter contract
+In an editor, modify the Counter contract to change the calculation in the `increaseCounter` function to add 3.
+
+[source,solidity]
+----
+  function increaseCounter(uint256 amount) public {
+    count = count + 3;
+  }
+----
+
+The Solidity Hot Loader will recompile the contract and upgrade it on the development network and update the artifacts. We can see in the terminal when it is compiling.
+
+In the app, notice that the state doesn't change. Then press *Increase Counter by 1* button. Confirm the transaction in MetaMask and once the transaction is confirmed the app will have the updated counter value. 
+The value should increase by 3.
+
+== Review
+
+Let's take a moment and review the Solidity Hot Loader development cycle: 
```

Comment:
I'd avoid repeating the content as-is from the overview (though the underlying idea is great)

Reply:
Removed review section